src/audio/al-util.h

```diff
@@ -168,6 +169,9 @@ namespace Source
 
 	inline void setVolume(Source::ID id, float value)
 	{
+		if (value > FLT_EPSILON) {
+			value = std::pow(10.0f, -(35.0f / 20.0f) * (1.0f - value));
```

Comment:
Can you add a comment here, that explains how you reverse-engineered the Enterbrain runtime to determine that this is correct? It would make it a lot easier for anyone in the future to understand what this code is doing.

Reply:
Done